src/fuzzer.rs

```diff
@@ -336,9 +336,15 @@ pub fn fuzz() -> Result<()> {
         .map(|timeout| Instant::now() - starting_time < timeout)
         .unwrap_or(true)
     {
-        fuzzer
-            .fuzz_one(&mut stages, &mut executor, &mut state, &mut mgr)
-            .context("Error in the fuzzing loop")?;
+        match fuzzer.fuzz_one(&mut stages, &mut executor, &mut state, &mut mgr) {
+            Ok(res) => res,
+            Err(libafl_bolts::Error::ShuttingDown) => {
+                return Ok(());
+            }
+            _ => {
+                panic!("Error in the fuzz loop.");
```

Comment:
Any reason you replaced the `?` (returns Err) by a panic?

Reply:
Wasn't on purpose, either is fine by me. The Err results in a hang in the fuzzer because of the executor that we use. Not sure what a panic does..